RNTester/RNTester-macOS/AppDelegate.mm

```diff
@@ -71,6 +78,31 @@ - (NSURL *)sourceURLForBridge:(__unused RCTBridge *)bridge
                                                         fallbackResource:nil];
 }
 
+#pragma mark - RCTCxxBridgeDelegate Methods
+
+- (std::unique_ptr<facebook::react::JSExecutorFactory>)jsExecutorFactoryForBridge:(RCTBridge *)bridge
+{
+  __weak __typeof(self) weakSelf = self;
+  return std::make_unique<facebook::react::JSCExecutorFactory>([weakSelf, bridge](facebook::jsi::Runtime &runtime) {
+    if (!bridge) {
```

Comment:
This is copied from RNTester/AppDalegate.mm, I think the reason it was written in this way could be, prevent the arguments from being nil and crash the program. But semantically it is required to be non-nil

Reply:
In that case it might be better to document it as such by marking the argument with `nonnull`?

Still, and this is mostly just a conceptual FYI, theoretically the bridge could have been disposed in-between the request to take a screenshot and actually taking it, so it might be better to capture `bridge` weakly here.